src/AWSMSKAuthTokenGenerator.cs

```diff
@@ -1,225 +1,306 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-
-using Amazon;
-using Amazon.Runtime;
-using Amazon.Runtime.CredentialManagement;
-using Amazon.Runtime.Endpoints;
-using Amazon.Runtime.Internal;
+﻿using Amazon.Runtime.CredentialManagement;
```

Comment:
Could you leave the copyright/license header?

Reply:
Sorry It was removed automatically from the cleanup.